packages/react-dom/src/events/BeforeInputEventPlugin.js

```diff
@@ -229,7 +243,7 @@ function extractCompositionEvent(
     return null;
   }
 
-  if (useFallbackCompositionData) {
+  if (useFallbackCompositionData && !isUsingKoreanIME(nativeEvent)) {
```

Comment:
Is there any reason you don't make this check a part of `useFallbackCompositionData` declaration itself?

Reply:
Since my implementation of `isUsingKoreanIME` depends on `CompositionEvent.locale` property from composition event, making it part of `useFallbackCompositionData` requires it to be a function that needs to be invoked every time a composition event occurs.

Current declaration of `useFallbackCompositionData` happens at load time as it only checks for browser. Check for Korean IME only needs to be invoked when using fallback mode, so I thought additional function invocation would be inefficient for those with modern browsers, though it will be certainly better for code readability.